libs/core/src/components/pds-input/pds-input.scss

```diff
@@ -17,6 +17,10 @@
   flex-direction: column;
 }
 
+label {
```

Comment:
It is not a blocker, but this component uses classes for most other element styling. Should we do the same for the label?